packages/stratocacher/src/wrap.js

```diff
@@ -73,6 +74,18 @@ export default function wrap(opts, func){
 				});
 		}
 
+		const set = (layers, val) => {
```

Comment:
Seems like this could be simpler if it just took a single layer.

Then `set(layer, val)` would be a drop-in replacement wherever `layer.set(val)` was used previously.